makefile

```diff
@@ -11,6 +23,7 @@ clean: static_clean
 	       .npm_installed \
 	       .peep_installed
 	find . -name "*.pyc" -exec rm -f {} \;
+	docker rm -f dxr_index dxr_serve dxr_test || true
```

Comment:
If any of the containers are not present, then it goes to `true` branch.

I've been using `make clean` after running tests, or after making an index-serve of code in docker/ and want to clear out all of my containers because I have changed DXR locally and need to rebuild the image anyway.